github/filecoin-project.yml

```diff
@@ -3420,8 +3336,6 @@ repositories:
         - proofs-crate-owners
       maintain:
```

Comment:
On all other repos, `proofs` has only push access. I don't think `merkletree` should be a special case here.